includes/frontend.php

```diff
@@ -1163,11 +1163,19 @@ protected function get_init_settings() {
 
 			$title = Utils::urlencode_html_entities( wp_get_document_title() );
 
+			// Try to get the 'large' size featured image thumbnail
```

Comment:
Please explain why use the `large`.

Reply:
Because it is the largest default WP size - the chances someone would change the dimensions of it are very small. It is the safest choice if we want to avoid using an expensive operation to find the largest image that would pass through Pinterest.

Anyway, I am also e-mailing with Pinterest showing them the issue, hopefully we will find the culprit.

Reply:
I mean explain in code (or maybe close the PR until it's ready for merge?)

Reply:
Ok will do

Reply:
Would rather not close it to not forget about it.
Marked it as WIP, this should prevent merge

Reply:
Done